### DISCUSSION TITLE
v1.8.1 Crashes the General Setting Tab

### DISCUSSION THREAD

**Discussion by TonyStark484** [deniscerri/ytdlnis#643] [Q&A] [answered]
- I just updated to the v1.8.1 from v.1.8.0. Since the settings are completely arranged differently this **breaks/crashes** the **general** tab in settings. A temporary fix would be to reset the saved settings or to clean install. But, after clean install if you try to use the backup json file to restore settings, it again breaks/crashes the **general** tab in settings.

**Comment by error-reporting** [comment]
It works for me

**Comment by TonyStark484** [comment]
> It works for me

Idk it restored the settings and everything but after that if you I click on the **General** tab in settings, the app crashes... I still don't know the cause. But, I literally had to reconfigure all the settings manually using 1.8.0 Backup file.

**Reply by error-reporting** [reply]
don't backup and restore settings when update from ytdlnis 1.8.0 to 1.8.1

**Reply by TonyStark484** [reply]
> don't backup and restore settings when update from ytdlnis 1.8.0 to 1.8.1

What? I don't understand, my point is the backup file becomes useless since the settings are revamped. It only breaks the General Tab making it unable to access. The only way is to Manually set the options instead of using a backup file to restore which makes the whole process tedious.

**Reply by deniscerri** [reply]
Someone gave me an error log but that log was useless. So i cant find out which preference is ruining things.

**Reply by TonyStark484** [reply]
> Someone gave me an error log but that log was useless. So i cant find out which preference is ruining things.

![YTDLnis_1.8.0_2024-11-27 [282.json.txt](https://github.com/user-attachments/files/17962459/YTDLnis_1.8.0_2024-11-27.282.json.txt)

I'm uploading my backup .json file for troubleshooting.

Just remove the .txt extension and try to use this backup file if it helps resolve the problem. I still couldn't use this backup file as it ends up breaking the General Tab as I mentioned earlier.

**Reply by deniscerri** [reply, marked as answer]
I have fixed the issue. For some reasons the Hide Thumbnails preference crashed the app because it couldnt produce a summary for the value. Even though its an empty array it shouldnt even hit that code, it somehow did. Did some extra checks and its resolved. I guess i need to release 1.8.2 earlier lol
